docs/content/Getting-Started/user-quickstart.md

```diff
@@ -0,0 +1,253 @@
+         kubectl create namespace kubestellar
+         helm repo add kubestellar https://helm.kubestellar.io
+         helm install kubestellar/kubestellar-core --set EXTERNAL_HOSTNAME="localhost" --set EXTERNAL_PORT={{ config.ks_port_num }} --namespace kubestellar --generate-name
+         ```
+    === "when is KubeStellar ready?"
+         run the following to wait for KubeStellar to be ready to take requests:
+         ```
+         echo -n 'Waiting for KubeStellar to be ready'
+         while ! kubectl exec $(kubectl get pod --selector=app=kubestellar -o jsonpath='{.items[0].metadata.name}' -n kubestellar) -n kubestellar -c init -- ls /home/kubestellar/ready &> /dev/null; do
+            sleep 10
+            echo -n "."
+         done
+         echo "KubeStellar is now ready to take requests"
+         ```
+    === "uh oh, error?"
+         Checking the initialization log to see if there are errors:
+         ```
+         kubectl logs $(kubectl get pod --selector=app=kubestellar -o jsonpath='{.items[0].metadata.name}' -n kubestellar) -n kubestellar -c init
+         ```
+         if there is nothing obvious, [open a bug report and we can help you out](https://github.com/kubestellar/kubestellar/issues/new?assignees=&labels=kind%2Fbug&projects=&template=bug_report.yaml&title=bug%3A+)
+    
+    === "open a bug report"
+        Stuck? [open a bug report and we can help you out](https://github.com/kubestellar/kubestellar/issues/new?assignees=&labels=kind%2Fbug&projects=&template=bug_report.yaml&title=bug%3A+)
+
+#### 2. Install KubeStellar's user commands and kubectl plugins
+
+!!! tip ""
+    === "install"
+         ```
+         brew tap kubestellar/kubestellar
+         brew install kcp_cli
+         brew install kubestellar_cli
+         ```
+    === "remove"
+         ```
+         brew remove kubestellar_cli
+         brew remove kcp_cli
+         brew untap kubestellar/kubestellar
+         ```
+
+
+#### 3. View your KubeStellar Core Space environment
+!!! tip ""
+    === "show all available spaces"
+         ```
+         kubectl get secrets kubestellar -o jsonpath='{.data.external\.kubeconfig}' -n kubestellar | base64 -d > ks-core.kubeconfig
+         KUBECONFIG=ks-core.kubeconfig kubectl ws --context root tree
+         ```
+    === "uh oh, error"
+         Did you received the following error:
+         ```Error: Get "https://some_hostname.some_domain_name:{{config.ks_port_num}}/clusters/root/apis/tenancy.kcp.io/v1alpha1/workspaces": dial tcp: lookup some_hostname.some_domain_name on x.x.x.x: no such host``
+
+         A common error occurs if you set your port number to a pre-occupied port number and/or you set your EXTERNAL_HOSTNAME to something other than "localhost" so that you can reach your KubeStellar Core from another host, check the following:
+         
+         Check if the port specified in the ks-core kind cluster configuration and the EXTERNAL_PORT helm value are unoccupied by another application:
+
```

Comment:
"..are unoccupied by..." isn't it "Check if... are occupied by another application." ?